examples/mugen/requirements.txt

```diff
@@ -1 +1,2 @@
+pytest-mock==3.8.2
```

Comment:
since this could be general too all our unit tests, would move to `dev-requirements.txt`